.changeset/yellow-eagles-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+chore!: renamed `AssetId` to `TestAssetId`
```

Comment:
Great!

Is this still a draft?

Reply:
It is not now - do you have a suggested title?

Reply:
Nope. 👍